app/src/main/java/org/sil/storyproducer/controller/export/ShareActivity.kt

```diff
@@ -82,6 +83,34 @@ class ShareActivity : PhaseBaseActivity(), RefreshViewListener {
             mNoVideosText!!.visibility = View.GONE
         }
         videosAdapter!!.setVideoPaths(exportedVideos)
+
+        mOpenVideoPath.setOnClickListener {
+
+            val videoContentUri  = getWorkspaceUri("$VIDEO_DIR/")
+            var videoFileUriStr = UriUtils.getPathFromUri(this, videoContentUri!!)
+
+            // At this point the videoFileUriStr will look something like this: /storage/emulated/0/
+            // This is the actual path. However, it needs be changed to the SD Card (/sdcard/)
+            // which is a symbolic link to the emulated storage path.
+            // sdcard/: Is a symlink to...
+            //      /storage/sdcard0 (Android 4.0+)
+            // In Story Producer, the version will never be less than Android 4.0
+            // We will instead show it as an optional [sdcard]
+            // The below code will change: /storage/emulated/0/ to /storage/[sdcard]/
+            videoFileUriStr = videoFileUriStr.replace(Regex("(storage\\/emulated\\/)\\d+"), "storage/[sdcard]")
+
+            // Also, the SD-Card could show up as /storage/####-####/ where # is a hexidecimal value
+            videoFileUriStr = videoFileUriStr.replace(Regex("(storage)\\/[0-9a-fA-F]{4}-[0-9a-fA-F]{4}"), "storage/[sdcard]")
+
+            var videoFileUri = Uri.parse(videoFileUriStr)
+
+            val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+            builder.setMessage("Using a file manager app, video files can be accessed at: ${videoFileUri.path}")
```

Comment:
Hardcoded strings will need to be moved to where they can be properly localized.

Reply:
@bryanwussow Good catch. Just fixed the hardcoded strings!